lib/location/cloud_service/cloud_service_nrf.c

```diff
@@ -101,17 +102,17 @@ int cellular_nrf_cloud_pos_get(
 	struct nrf_cloud_rest_context rest_ctx = {
 		.connect_socket = -1,
 		.keep_alive = false,
-		.timeout_ms = params->timeout,
+		.timeout_ms = params->timeout_ms,
 		.rx_buf = rcv_buf,
 		.rx_buf_len = rcv_buf_len,
 		.fragment_size = 0
 	};
 	const struct nrf_cloud_rest_location_request loc_req = {
 		.cell_info = (struct lte_lc_cells_info *)params->cell_data,
```

Comment:
Is casting needed here to remove `const`? That's not very nice. Couldn't we remove the `const` keyword from `struct cloud_service_pos_req` instead?

Reply:
Seems this has been for the `const`. We can always remove them :) I guess here it's fine.

Reply:
I think it should be changed, because casting constness away is bad practice. If there's a good reason for something to be `const`, you shouldn't be able to modify the data. Here I don't think there's a real reason for the data to be `const`, so it's better to change it "at the source". Or the functions which use the data could be changed to accept `const`, but that's easier said than done even if the functions do not actually need to modify the data.